### PR TITLE
[docs][deploy] Format and links

### PR DESCRIPTION
**Pull request by ddhodge**
Fix formatting and relative links

@netlify /preview/deploy/public-clouds/

### CODE REVIEW

File: docs/content/preview/deploy/public-clouds/azure/azure-arm.md [hunk 1]
```diff
@@ -121,13 +123,13 @@ type: docs
...
 
-1. Once the deployment creation is complete, you can describe it as shown below.
+1. After the deployment creation is complete, you can describe it as shown below.
 
     ```sh
     $ az group deployment show -g <Your-Azure-Resource-Group> -n <Your-Deployment-Name> --query properties.outputs
     ```
 
-    In the output, you will get the YugabyteDB admin URL, JDBC URL, YSQL, YCQL, and YEDIS connection string. You can use YugabyteDB admin URL to access admin portal.
+    The output displays the YugabyteDB Admin URL, JDBC URL, YSQL, YCQL, and YEDIS connection string. You can use the YugabyteDB Admin URL to access Admin portal.
```
Comment (aishwarya24) [inline docs/content/preview/deploy/public-clouds/azure/azure-arm.md:132]:
```suggestion
    The output displays the YugabyteDB Admin URL, JDBC URL, YSQL, YCQL, and YEDIS connection string. You can use the YugabyteDB Admin URL to access the Admin portal.
```

File: docs/content/preview/deploy/public-clouds/gcp/gcp-deployment-manager.md [hunk 2]
```diff
@@ -38,27 +38,27 @@ type: docs
...
+* Wait 5-10 minutes for the creation of all resources to complete.
+* After the deployment creation is complete, you can describe it as follows:
 
-    ```
+    ```sh
     $ gcloud deployment-manager deployments describe <your-deployment-name>
     ```
 
-    In the output, you will get the YugabyteDB admin URL, JDBC URL, YSQL, YCQL and YEDIS connection string. You can use YugabyteDB admin URL to access admin portal.
+The output includes the YugabyteDB admin URL, JDBC URL, and YSQL, YCQL, and YEDIS connection strings. You can use the YugabyteDB admin URL to access the admin portal.
```
Comment (aishwarya24) [inline docs/content/preview/deploy/public-clouds/gcp/gcp-deployment-manager.md:64]:
```suggestion
The output includes the YugabyteDB Admin URL, JDBC URL, and YSQL, YCQL, and YEDIS connection strings. You can use the YugabyteDB Admin URL to access the Admin portal.
```

File: docs/content/preview/deploy/public-clouds/gcp/terraform.md [hunk 3]
```diff
@@ -59,103 +61,110 @@ Common commands:
...
+## Create a cluster
+
+Init terraform first if you haven't already done so.
 
 ```sh
 $ terraform init
 ```
 
-To check what changes are going to happen in environment run the following
+To check what changes are going to happen in environment run the following:
```
Comment (aishwarya24) [inline docs/content/preview/deploy/public-clouds/gcp/terraform.md:115]:
```suggestion
To check what changes are going to happen in your environment, run the following:
```

File: docs/content/preview/deploy/public-clouds/gcp/terraform.md [hunk 4]
```diff
@@ -59,103 +61,110 @@ Common commands:
...
-- `module.terraform-gcp-yugabyte.google_compute_instance.yugabyte_node` The GCP VM instances.
+- `module.terraform-gcp-yugabyte.google_compute_instance.yugabyte_node`
+
+    The GCP VM instances.
+
+    For a cluster named `test-cluster`, the instances are named `yugabyte-test-cluster-n1`, `yugabyte-test-cluster-n2`, and `yugabyte-test-cluster-n3`.
+
+- `module.terraform-gcp-yugabyte.google_compute_firewall.Yugabyte-Firewall`
+
+    The firewall rule that allows the various clients to access the YugabyteDB cluster.
```
Comment (aishwarya24) [inline docs/content/preview/deploy/public-clouds/gcp/terraform.md:151]:
```suggestion
    The firewall rule that allows various clients to access the YugabyteDB cluster.
```

**Review by aishwarya24 — COMMENTED** [review]
Reviewed preview changes. Looking into stable and others next.

File: docs/content/preview/deploy/public-clouds/gcp/gcp-deployment-manager.md [hunk 5]
```diff
@@ -38,27 +38,27 @@ type: docs
 
 ## Prerequisites
 
-* Download and Install [gcloud](https://cloud.google.com/sdk/docs/) command line tool.
-* Clone git repo from [here](https://github.com/yugabyte/gcp-deployment-manager.git)
+* Download and Install [Google Cloud CLI](https://cloud.google.com/sdk/docs/).
+* Clone the [Google Cloud Deployment Manager for YugabyteDB](https://github.com/yugabyte/gcp-deployment-manager.git) repository.
 
-## Deploying using Cloud Shell
+## Deploy using Cloud Shell
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/deploy/public-clouds/gcp/gcp-deployment-manager.md:44]: To be fixed with security policy fix

**Review by aishwarya24 — COMMENTED** [review]
Verified links for rest of the versions. My earlier comments would apply to other versions too.

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Pull request merged**